templates/bin_setup.erb

```diff
@@ -7,8 +7,8 @@
 set -e
 
 # Set up Ruby dependencies via Bundler
-gem list bundler --installed > /dev/null || gem install bundler
-bundle install
+command -v bundle > /dev/null || gem install bundler
```

Comment:
Ah, good to know.  I cargo-culted it from lower down in the script.  

I can change the PR to then 

1. add `bundle check`
2. apply the regex fix to `gem list`

@jferris suggested `gem install bundler --conservative` but I think that is unrelated to this PR, which was originally just about `bundle check`'ing "all the things" :)